maturity-matrix.yaml

```diff
@@ -47,9 +47,11 @@ productionMaturityLevels:
   # typical production use cases.
   - stable
 
-# All OpenTelemetry APIs follow semver conventions (i.e.,
-# backwards-incompatibly changes should bump the major version number).
+# All OpenTelemetry APIs follow semver conventions (i.e., after v1.x,
```

Comment:
yes, thank you for this! that's actually what I'd meant to write, so it's wonderful to see your telepathy skills on display... :)